indico/core/config.py

```diff
@@ -276,102 +276,90 @@ class Config:
-                             os.path.join("meetings", "IndicoMeeting.tpl"),
+                             'meeting.html',
                              os.path.join("meetings", "LCG.css")),
         "lhcb_meeting":     ("LHCb meeting",
-                             os.path.join("meetings", "IndicoMeeting.tpl"),
+                             'meeting.html',
                              os.path.join("meetings", "LHCb.css")),
         "egee_meeting":     ("EGEE meeting",
-                             os.path.join("meetings", "IndicoMeeting.tpl"),
+                             'meeting.html',
                              os.path.join("meetings", "EGEE.css")),
         "cms":              ("CMS meeting",
-                             os.path.join("meetings", "IndicoMeeting.tpl"),
+                             'meeting.html',
                              os.path.join("meetings", "CMS.css")),
         "endotofpet":       ("EndoTOFPET",
-                             os.path.join("meetings", "Openlab.tpl"),
+                             'openlab.html',
                              os.path.join("meetings", "EndoTOFPET.css")),
         "crystal_clear":    ("Crystal Clear",
-                             os.path.join("meetings", "Openlab.tpl"),
+                             'openlab.html',
                              os.path.join("meetings", "CrystalClear.css")),
         "openlab":          ("Openlab",
-                             os.path.join("meetings", "Openlab.tpl"),
+                             'openlab.html',
                              os.path.join("meetings", "Openlab.css")),
 
-        # CDS Agenda styles
-        "cdsagenda":        ("CDS Agenda style",
-                             os.path.join("meetings", "cdsagenda", "CDSAgenda.tpl"),
-                             os.path.join("meetings", "cdsagenda", "CDSAgenda.css")),
-        "cdsagenda_inline_minutes":
-                            ("CDS Agenda inline minutes",
-                             os.path.join("meetings", "cdsagenda", "CDSAgendaWithMinutes.tpl"),
-                             os.path.join("meetings", "cdsagenda", "CDSAgendaWithMinutes.css")),
-        "cdsagenda_olist":  ("CDS Agenda ordered list",
-                             os.path.join("meetings", "cdsagenda", "CDSAgendaOrdered.tpl"),
-                             os.path.join("meetings", "cdsagenda", "CDSAgendaOrdered.css")),
-
         # Standard conference style
         "static":           ("Parallel", None, None),
 
         # Other meeting styles
         "nicecompact":      ("Compact style",
-                             os.path.join("meetings", "Compact.tpl"),
+                             'compact.html',
                              os.path.join("meetings", "Compact.css")),
-        "weeks":            ("Compact weeks",
-                             os.path.join("meetings", "Weeks.tpl"),
-                             os.path.join("meetings", "Weeks.css")),
+        "indico-weeks-view": ("Compact weeks",
```

Comment:
Is it a good idea to rename this? It would break for people linking to that specific view...